omod/src/main/webapp/resources/scripts/fragments/patientdashboard/encountertemplate/defaultEncounterTemplate.js

```diff
@@ -14,7 +14,7 @@ $(function() {
 		showDeleteEncounterDialog();
 	});
 
-    $(document).on('click', '.editEncounter', function(event) {
+    $(document).on('click', '.editEncounter, .viewEncounter', function(event) {
```

Comment:
@dkayiwa This is to make sure that the objects with viewEncounter class respond to the clicks.